src/content/en/updates/2017/02/media-session.md

```diff
@@ -0,0 +1,374 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Customize media metadata (artist, album, title, artwork) and respond to media controls (play, pause, etc.) on the Web.
+
+{# wf_updated_on: 2017-02-06 #}
+{# wf_published_on: 2017-02-06 #}
+{# wf_tags: news,mediasession,play,pause #}
```

Comment:
Build Errors for uncommon tags. You can add these tags in `src/data/commonTags.json`. Be sure to review the file first to make sure you're not creating similar tags.
* [ ] src/content/en/updates/2017/02/media-session.md Uncommon tag (mediasession) found.
* [ ] src/content/en/updates/2017/02/media-session.md Uncommon tag (pause) found.